portfolio/src/main/webapp/style.css

```diff
@@ -119,3 +119,69 @@ h2 {
   margin-bottom: 30px;
 }
 
+#random {
+  background-color: oldlace;
+  padding: 10vh 15vw;
+  margin-top: -5px;
+}
+
+#prompt {
+  color: saddlebrown;
+  font: 20px Verdana, Geneva, sans-serif;
+  height: 30px;
+  margin: 20px 0px 15px;
+  width: 100%;
+}
+
+#prompt>p {
```

Comment:
nit: I can't find anything on the style guide about this, but I think typically we put space around the greater than sign: `#prompt > p` e.g. https://css-tricks.com/almanac/selectors/c/child/